lib/fs/pci.js

```diff
@@ -52,11 +52,15 @@ class Tree {
     const codeDep = getCodeDep(codeCommune)
     const directory = join(this.getFeuillesBasePath(), codeDep, codeCommune)
     const entries = await readdir(directory)
+    const regex = this.format === 'tiff' ?
+      /^tiff-([A-Z0-9]{12})\.zip$/i :
```

Comment:
Si c'est autre chose que tiff ou edigeo je préfèrerai qu'on throw une erreur.
`new Error('Not supported format')`

Reply:
`listFeuillesByCommune()` est aussi utilisé pour le `pci-vecteur` donc le format peut varier.

Reply:
Ça n'empêche pas de sortir une erreur :)

Reply:
Ce n'est pas déjà ce que fait `validFormat` ?

Reply:
Certes mais ce code à vocation à être backporté dans cadastre.